api/src/main/java/org/openmrs/module/kenyaemr/reporting/cohort/definition/evaluator/HTSFamilyContactsTestedCohortDefinitionEvaluator.java

```diff
@@ -47,7 +47,7 @@ public EvaluatedCohort evaluate(CohortDefinition cohortDefinition, EvaluationCon
 
 		Cohort newCohort = new Cohort();
 
-		String qry="select c.id from kenyaemr_etl.etl_hts_contacts c where c.relationship_type in (970,971,972,1528,5617,162221)\n" +
+		String qry="select c.id from kenyaemr_etl.etl_hts_contacts c where c.relationship_type in (3,2,6,8)\n" +
```

Comment:
We can join this to the relationship type and use the UUIDs. I am not sure if that will be unique though

Reply:
Sure, it should be more safe to use uuids. Will make the changes across are refs.